package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "oio-react",
+  "version": "0.0.1",
+  "description": "OIO for React",
+  "scripts": {
+    "build": "NODE_ENV=production rm -rf dist && webpack --config config/webpack/build.js",
+    "demo": "NODE_ENV=development node ./demo/server",
+    "lint": "git diff HEAD --name-only --diff-filter=ACM | grep '.js$' | xargs node ./node_modules/eslint/bin/eslint.js --quiet",
+    "lint-full": "node ./node_modules/eslint/bin/eslint.js .",
+    "prettier": "prettier --write src/OIO/index.js",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "webpack-analyze": "mkdir -p dist; NODE_ENV=production webpack --config config/webpack/build.js --profile --json > dist/stats.json; webpack-bundle-analyzer dist/stats.json"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint && npm audit",
+      "post-merge": "npm install",
+      "post-receive": "npm install",
+      "post-rewrite": "npm install"
+    }
+  },
+  "main": "dist/index.js",
+  "keywords": [
+    "OIO"
+  ],
+  "author": "Mother Co.",
+  "license": "MIT",
+  "peerDependencies": {
+    "react": "^16.3.0",
+    "react-dom": "^16.3.0"
+  },
+  "devDependencies": {
+    "@babel/core": "7.1.2",
+    "@babel/plugin-proposal-class-properties": "7.1.0",
+    "@babel/plugin-proposal-export-default-from": "7.0.0",
```

Comment:
As discussed, I would not use this plugin, as it is still a Stage 0 proposal within the tc39 process (as opposed to class properties, which is a stage 3 proposal)